integration-tests/test_client.py

```diff
@@ -97,3 +97,28 @@ def test_verify_logrotate_feature(insights_client):
     subprocess.check_call(["logrotate", "-vf", logrotate_conf_file_path])
     number_of_files_after_logrotate = len(os.listdir(logdir))
     assert number_of_files_after_logrotate == (number_of_log_files + 1)
+
+
+@pytest.mark.usefixtures("register_subman")
+def test_insights_directory_files(insights_client):
+    """
+    Test that /var/lib/insights have the right files available
+    """
+    directory = "/var/lib/insights"
+    registered_contents = [
+        "last_stable.egg",
+        "last_stable.egg.asc",
+    ]
```

Comment:
Note that `newest.egg` (+ `.asc`) are also files that may show up in that directory.

Note²: it seems the "malware" application in insights-core stores `.json` cache files in that directory.

Reply:
solved